bundles/org.openhab.binding.surepetcare/src/main/resources/OH-INF/thing/things.xml

```diff
@@ -0,0 +1,619 @@
+	</channel-type>
+
+	<channel-type id="curfewUnlockTimeType">
+		<item-type>String</item-type>
+		<label>Curfew Unlock Time</label>
+		<description>The time when the curfew finishes</description>
+		<state readOnly="false" pattern="%s"/>
+	</channel-type>
+
+	<channel-type id="dateOfBirthType">
+		<item-type>DateTime</item-type>
+		<label>Pet Date of Birth</label>
+		<description>The pet's date of birth.</description>
+		<category>Time</category>
+		<state readOnly="true" pattern="%1$tY-%1$tm-%1$td"/>
+	</channel-type>
+
+	<channel-type id="weightType">
+		<item-type>Number:Mass</item-type>
+		<label>Pet Weight</label>
+		<description>The pet weight</description>
+		<state readOnly="true" pattern="%.1f %unit%"/>
+	</channel-type>
+
+	<channel-type id="feederDeviceType">
+		<item-type>String</item-type>
+		<label>Pet Feeding Device Name</label>
+		<description>The pet feeding device name</description>
+		<state readOnly="true" pattern="%s"/>
+	</channel-type>
+
+	<channel-type id="feederLastFeedingType">
+		<item-type>DateTime</item-type>
+		<label>Pet Last Feeding</label>
+		<description>The last pet feeding</description>
+		<state readOnly="true" pattern="%1$tY-%1$tm-%1$td %1$tH:%1$tM:%1$tS"/>
+	</channel-type>
+
+	<channel-type id="feederLastChangeType">
+		<item-type>Number:Mass</item-type>
+		<label>Pet Feeding Last Change</label>
+		<description>The pet feeding last change</description>
+		<state readOnly="true" pattern="%.2f %unit%"/>
+	</channel-type>
+
+	<channel-type id="feederLastChangeLeftType">
+		<item-type>Number:Mass</item-type>
+		<label>Pet Feeding Last Change Left</label>
+		<description>The pet feeding last change left</description>
+		<state readOnly="true" pattern="%.2f %unit%"/>
+	</channel-type>
+
+	<channel-type id="feederLastChangeRightType">
+		<item-type>Number:Mass</item-type>
+		<label>Pet Feeding Last Change Right</label>
+		<description>The pet feeding last change right</description>
+		<state readOnly="true" pattern="%.2f %unit%"/>
+	</channel-type>
+
+	<channel-type id="lockingModeType">
+		<item-type>String</item-type>
+		<label>Locking Mode</label>
+		<description>The id of the locking mode the flap is currently set to</description>
+		<state readOnly="false">
+			<options>
+				<option value="0">Unlocked</option>
+				<option value="1">Locked In-only</option>
+				<option value="2">Locked Out-only</option>
+				<option value="3">Locked</option>
+				<option value="4">Curfew</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="batteryVoltageType">
+		<item-type>Number</item-type>
+		<label>Battery Voltage</label>
+		<description>The current battery voltage</description>
+		<state readOnly="true" pattern="%.1f V"/>
+	</channel-type>
+
+	<channel-type id="rssiDeviceType" advanced="true">
+		<item-type>Number</item-type>
+		<label>Signal Strength Device (RSSI)</label>
+		<description>The received device signal strength (RSSI).</description>
+		<state readOnly="true" pattern="%.2f dB"/>
```

Comment:
Except for RSSI, that one is a bit funky so you can keep it as a raw number for now.